web/crux/src/app/team/team.service.ts

```diff
@@ -113,17 +115,23 @@ export class TeamService {
     const userIds: string[] = team.users.map(it => it.userId).concat(team.invitations.map(it => it.userId))
     const identities = await this.kratos.getIdentitiesByIds(userIds)
 
-    return this.mapper.detailsToGrpc(team, identities)
+    return this.mapper.activeTeamDetailsToGrpc(team, identities)
   }
 
   async createTeam(
     request: CreateTeamRequest,
     call: ServerUnaryCall<CreateTeamRequest, Promise<CreateEntityResponse>>,
   ): Promise<CreateEntityResponse> {
-    const userHasTeam = await this.teamRepository.userHasTeam(request.accessedBy)
+    const hasActiveTeam = await this.prisma.usersOnTeams.findFirst({
+      rejectOnNotFound: false,
+      where: {
+        userId: request.accessedBy,
+        active: true,
+      },
+    })
 
-    // If user doesn't has team, make the current one to active
-    const active = !userHasTeam
+    // If user doesn't has active team, make the current one to active
```

Comment:
`If the user doesn't have an active team, make the current one active` seems better I think.